diff --git a/node_modules/@figma/plugin-typings/index.d.ts b/node_modules/@figma/plugin-typings/index.d.ts
index bdc53d0..d260677 100644
--- a/node_modules/@figma/plugin-typings/index.d.ts
+++ b/node_modules/@figma/plugin-typings/index.d.ts
@@ -7,7 +7,6 @@ declare global {
   const __uiFiles__: {
     [key: string]: string
   }
-  const console: Console
 
   // The plugin environment exposes the browser console API,
   // so expected calls like console.log() still work.
@@ -24,8 +23,6 @@ declare global {
   function setInterval(callback: Function, timeout: number): number
   function clearInterval(handle: number): void
 
-  const fetch: (url: string, init?: FetchOptions) => Promise<FetchResponse>
-
   interface FetchOptions {
     method?: string
     headers?: { [name: string]: string }
